docs/ENV.MD

```diff
@@ -5,4 +5,17 @@ This guide explains all the available configurations in your `.env` file so that
 API key for third-party access, given to each organization. To know more about custom authentication, click [here].(https://docs.plio.in/plio-for-teams/#custom-authentication)    
 
 ## VUE_APP_BASE_URL_PLIO=''
-Base URL for the Plio link.
\ No newline at end of file
+Base URL for the Plio link.
+
+## VUE_APP_AWS_SQS_ACCESS_KEY=''
+AWS access key for access to AWS SQS.
+
+## VUE_APP_AWS_SQS_SECRET_ACCESS_KEY=''
+AWS secret key for access to AWS SQS.
+
+## AWS_S3_ACCESS_KEY_ID=''
+AWS access key for access to AWS S3.
+
+## AWS_S3_SECRET_ACCESS_KEY=''
+AWS secret key for access to AWS S3.
```

Comment:
These two things don't go into `.env` file right?
Ideally, both of these env variables should be mentioned in the DEPLOYMENT.md or you can also put it in INSTALLATION.md

Reply:
Actually, you're right. The SQS secrets should be in INSTALLATION but the S3 should be in DEPLOYMENT.

Reply:
Actually, you're right. The SQS secrets should be in INSTALLATION but the S3 should be in DEPLOYMENT.